sw_app/pubspec.yaml

```diff
@@ -50,7 +50,8 @@ dev_dependencies:
   # package. See that file for information about deactivating specific lint
   # rules and activating additional ones.
   flutter_lints: ^2.0.0
-  provider: ^6.0.0
+  #provider: ^6.0.0
+  path_provider: ^2.0.6
```

Comment:
Could you update the README accordingly:
https://github.com/andreped/DSS/blob/main/README.md

As provider has been replaced by path_provider it should be in the list of dependencies instead.

Reply:
Fixed in latest commit